_docs-sources/pipelines/security/audit-log.md

```diff
@@ -0,0 +1,116 @@
+
+Pipelines uses a naming scheme that combines the GitHub user who triggered the [action](../overview/actions.md) and the pull request or branch from which the action was initiated. There are two scenarios in which Pipelines runs actions - when a pull request is opened, synchronized, or re-opened, and when a pull request is merged to the deploy branch (e.g., `main`).
+
+When Pipelines is run in response to a pull request event, the user that created the most recent commit on the branch will be used in the log, in addition to the pull request number assigned by GitHub. For example, if the user `SomeUserInYourOrg123` created a pull request that was given the number `123` by GitHub, the resulting session name would be `SomeUserInYourOrg123-via-GWPipelines@PR-123`.
+
+When Pipelines is run in response to a pull request being merged, the user that performed the merge (e.g., the user who clicked the `merge` button on the pull request) will be used in the log in addition the to deploy branch (e.g., `main`). For example, if the user `SomeUserInYourOrg123` merged a pull request to your deploy branch named `main` the session name would be `SomeUserInYourOrg123-via-GWPipelines@main`
+
+### What gets logged
+
+Logs are available for all operations performed in every AWS account by Gruntwork Pipelines. Gruntwork Pipelines takes advantage of AWS STS session names to clearly label all sessions with the GitHub user who requested the change and the pull request or branch that triggered the change.
+
+The `userIdentity` field in each CloudTrail event associated with API calls performed by Pipelines [actions](../overview/actions.md) will contain the session name. For example, if the GitHub user `SomeUserInYourOrg123` made a pull request that was the 123rd pull request in your repository, an associated CloudTrail event would have the following data for `userIdentity`.
+
+```
+{
+    "eventVersion": "1.09",
+    "userIdentity": {
+        "type": "AssumedRole",
+        "principalId": "xxxxxxxxxxxxxxxxxxxxx:SomeUserInYourOrg123-via-GWPipelines@PR-123",
+        "arn": "arn:aws:sts::123456789012:assumed-role/github/SomeUserInYourOrg123-via-GWPipelines@PR-123",
+        "accountId": "123456789012",
+        "accessKeyId": "xxxxxxxxxxxx",
+        "sessionContext": {
+            "sessionIssuer": {
+                "type": "Role",
+                "principalId": "xxxxxxxxxxxxxxxxxxxxx",
+                "arn": "arn:aws:iam::123456789012:role/github",
+                "accountId": "123456789012",
+                "userName": "github"
+            },
+            "webIdFederationData": {
+                "federatedProvider": "arn:aws:iam::123456789012:oidc-provider/token.actions.githubusercontent.com",
+                "attributes": {}
+            },
+            "attributes": {
+                "creationDate": "2023-09-25T13:24:17Z",
+                "mfaAuthenticated": "false"
+            }
+        }
+    },
+    ... rest of the CloudTrail event here ...
+}
+```
+
```

Comment:
We should mention explicitly that we don't record the git repo due to the 64 character limit in the session identity but that it's something that can be easily inferred.

Reply:
Addressed in [e3f2fb2](https://github.com/gruntwork-io/docs/pull/1143/commits/e3f2fb2f2d7ff2fa02709cd3bd6cd2242c25f5be)